pkg/cmd/cli/install/install.go

```diff
@@ -130,6 +134,38 @@ func NewCommand(f client.Factory) *cobra.Command {
 	return c
 }
 
+func GetVeleroVersion(f client.Factory) (string, error) {
+	// Retrieve velero version on server side
+	serverStatusGetter := &serverstatus.DefaultServerStatusGetter{
+		Namespace: f.Namespace(),
+		Timeout:   5 * time.Second,
+	}
+	var veleroClient velerov1client.ServerStatusRequestsGetter
+	client_v, err := f.Client()
+	cmd.CheckError(err)
+	veleroClient = client_v.VeleroV1()
+	serverStatus, err := serverStatusGetter.GetServerStatus(veleroClient)
+	if err != nil {
+		return "", err
+	}
+	version := serverStatus.Status.ServerVersion
+    return version, nil
+}
+
+func CheckVeleroVersion(version string) bool {
+	var digits []string
+	if strings.Contains(version, "-") {
+		baseVersion := strings.Split(version, "-")[0]
+		digits = strings.Split(baseVersion, ".")
+	} else {
+		digits = strings.Split(version, ".")
+	}
+	if digits[0] == "v1" && (digits[1] == "0" || digits[1] == "1" || digits[1] == "2") {
```

Comment:
Why isn't this just digits[1] < MinVeleroMinorVersion (which should be "3") ?  Check the PointVersion here as well

Reply:
Same above.